libcxx/include/future

```diff
@@ -1460,8 +1460,10 @@ public:
   _LIBCPP_HIDE_FROM_ABI __packaged_task_function() _NOEXCEPT : __f_(nullptr) {}
   template <class _Fp>
   _LIBCPP_HIDE_FROM_ABI __packaged_task_function(_Fp&& __f);
+#  if _LIBCPP_STD_VER <= 14
```

Comment:
Since this is an implementation detail, we can provide it unconditionally IMO. We can guard only the public part of the API under a `_LIBCPP_STD_VER` check.

Reply:
Done.